cgroup.go

```diff
@@ -52,8 +52,9 @@ func Load(hierarchy Hierarchy, path Path) (Cgroup, error) {
 	if err != nil {
 		return nil, err
 	}
-	// check the the subsystems still exist
-	for _, s := range pathers(subsystems) {
```

Comment:
could we keep the `for range` the same and create a temporary var for valid subsystems?  Maybe something like `activeSubsystems` and you append active ones instead of removing the ones that are not found from the current list? I think it would make the code simpler and explicit.

What do you think?

Reply:
Sure! I had the same thought but chose to go with the more "economical" way by keeping the same array.